pytest/tests/mocknet/helpers/load_testing_add_and_delete_helper.py

```diff
@@ -0,0 +1,360 @@
+
+
+def get_latest_block_hash():
+    last_block_hash = get_status()['sync_info']['latest_block_hash']
+    return base58.b58decode(last_block_hash.encode('utf-8'))
+
+
+def send_transfer(account, i, node_account):
+    next_id = random.randrange(mocknet.NUM_ACCOUNTS)
+    dest_account_id = mocknet.load_testing_account_id(
+        node_account.key.account_id, next_id)
+    account.send_transfer_tx(dest_account_id)
+
+
+
+def function_call(account, i, node_account):
+    if random.randint(0,1) == 0:
+        s = f'{{"token_account_id": "token2.near"}}'
+        logger.info(
+            f'Calling function "withdraw_token" with arguments {s} on account {i}'
+        )
+        tx_res = account.send_call_contract_raw_tx(mocknet.SKYWARD_ACCOUNT, 'withdraw_token', bytes(s, encoding='utf-8'), 0)
+        logger.info(f'Account {account.key.account_id} withdraw_token {tx_res}')
+    else:
+        s = '{"sale_id": 0, "amount": "1"}'
+        logger.info(
+            f'Calling function "sale_deposit_in_token" with arguments {s} on account {i}'
+        )
+        tx_res = account.send_call_contract_raw_tx(mocknet.SKYWARD_ACCOUNT,
+                                          'sale_deposit_in_token',
+                                          bytes(s, encoding='utf-8'), 1)
+        logger.info(f'Account {account.key.account_id} sale_deposit_in_token {tx_res}')
+
+
+def skyward_transaction(account, i, node_account, max_tps_per_node):
+    time.sleep(random.random() * mocknet.NUM_ACCOUNTS / max_tps_per_node / 3)
+    function_call(account, i, node_account)
+
+
+def send_skyward_transactions(node_account, test_accounts, max_tps_per_node):
+    pmap(
+        lambda index_and_account: skyward_transaction(index_and_account[
+            1], index_and_account[0], node_account, max_tps_per_node),
+        enumerate(test_accounts))
+
+
+def throttle_txns(send_txns, total_tx_sent, elapsed_time, max_tps_per_node,
+                  node_account, test_accounts):
+    start_time = time.time()
+    send_txns(node_account, test_accounts, max_tps_per_node)
+    duration = time.time() - start_time
+    total_tx_sent += len(test_accounts)
+    elapsed_time += duration
+
+    excess_transactions = total_tx_sent - (max_tps_per_node * elapsed_time)
+    if excess_transactions > 0:
+        delay = excess_transactions / max_tps_per_node
+        elapsed_time += delay
+        logger.info(f'Sleeping for {delay} seconds to throttle transactions')
+        time.sleep(delay)
+
+    return (total_tx_sent, elapsed_time)
+
+
+def write_tx_events(accounts_and_indices, filename):
+    # record events for accurate input tps measurements
+    all_tx_events = []
+    for (account, _) in accounts_and_indices:
+        all_tx_events += account.tx_timestamps
+    all_tx_events.sort()
+    with open(filename, 'w') as output:
+        for t in all_tx_events:
+            output.write(f'{t}\n')
+
+
+def get_skyward_account():
+    global special_accounts
```

Comment:
No need for `global` here and below. You need `global` only if you bind the global variable to a different value.